lab-justin/test/note-route-get-test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const request = require('superagent');
+const expect = require('chai').expect;
+
+const port = process.env.PORT || 3000;
+const server = require('../server');
+const serverUrl = `http://localhost:${port}`;
+
+describe('testing note-route-two module', function(){
+
+  before(function(done) {
```

Comment:
By requiring the server in, you're always going to be starting it, so you don't really need all this isRunning stuff--your server isn't being shared across your test files, so the isRunning property on one won't be accessible in the others. If one is already running, it'll just break and tell you the address is already in use.